firebase.json

```diff
@@ -49,12 +49,6 @@
       { "source": "/assets-and-images", "destination": "/docs/development/ui/assets-and-images", "type": 301 },
       { "source": "/bootstrap-into-dart", "destination": "/docs/resources/bootstrap-into-dart", "type": 301 },
       { "source": "/bug-reports", "destination": "/docs/resources/bug-reports", "type": 301 },
-      { "source": "/catalog/samples", "destination": "/docs/catalog/samples", "type": 301 },
-      { "source": "/catalog/samples/tabbed-app-bar", "destination": "/docs/cookbook/design/tabs", "type": 301 },
-      { "source": "/catalog/samples/:page*_index", "destination": "/docs/catalog/samples/:page*", "type": 301 },
-      { "source": "/catalog/samples/AppBar_index", "destination": "/docs/catalog/samples/AppBar", "type": 301 },
-      { "source": "/catalog/samples/TabBar_index", "destination": "/docs/catalog/samples/TabBar", "type": 301 },
-      { "source": "/catalog/samples/:rest*", "destination": "/docs/catalog/samples/:rest*", "type": 301 },
```

Comment:
Do we want to keep a couple of /catalog/samples redirects (similar to the /docs/catalog/samples redirects) to forestall 404s? Or do you want to find out whether people are still using these URLs, even if it causes 404s?

Reply:
I'd rather wait and see.  :)

Reply:
More work for me. I see. 😄